transformers/embed.ts

```diff
@@ -42,12 +42,7 @@ export function transformEmbed(bot: Bot, payload: SnakeCasedPropertiesDeep<Embed
         width: payload.video.width,
       }
       : undefined,
-    provider: payload.provider
```

Comment:
Unrelated change please do this in a different PR.
I want to stick to the 1 PR 1 change schema.